### PR TITLE
19460 mark spent time as empty in case of 0 hours

### PR DESCRIPTION
**Pull request by 0xF013**
https://community.openproject.org/work_packages/19460

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-fix-19460-spent-time-empty-state.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-fix-19460-spent-time-empty-state.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by jonasheinrich** [comment]
@floriank I tried to help and got the following:
The behaviour is as wanted. When there is no logged time the field only appears when "show all" is clicked. When there is logged time it is always present and there is a correct link. 
Unfortunately, I cannot say anything about coding style whatsoever. 
But for me this looks good.

**Comment by floriank** [comment]
@jonasheinrich thanks! I will have a look at the code style.

File: frontend/app/work_packages/directives/inplace_editor/custom/display/inplace-display-spent-time-directive.js [hunk 1]
```diff
@@ -44,8 +44,9 @@ module.exports = function(EditableFieldsState) {
       };
     },
     controllerAs: 'customEditorController',
-    link: function(scope, element, attrs, displayPaneController) {
-      scope.displayPaneController = displayPaneController;
+    link: function(scope, element, attrs, controllers) {
+      scope.displayPaneController = controllers[0];
```
Comment (myabc) [inline frontend/app/work_packages/directives/inplace_editor/custom/display/inplace-display-spent-time-directive.js:48]: @0xF013 I want destructuring assignment already!

Comment (0xF013) [inline frontend/app/work_packages/directives/inplace_editor/custom/display/inplace-display-spent-time-directive.js:48]: @myabc I actually wrote something like (scope, element, attrs, [displayPaneController, fieldController]) then tried to understand what language I am hallucinating

**Pull request merged**